arboretum/permissions/checks/test_org_direct_collaborators.py

```diff
@@ -0,0 +1,115 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Github organization direct collaborators check."""
+
+import json
+
+from compliance.check import ComplianceCheck
+from compliance.evidence import DAY, ReportEvidence, get_evidence_by_path
+from compliance.utils.data_parse import get_sha256_hash
+
+
+class OrgCollaboratorsCheck(ComplianceCheck):
+    """Checks for organization repo direct collaborators."""
+
+    @property
+    def title(self):
+        """
+        Return the title of the checks.
+
+        :returns: the title of the checks
+        """
+        return 'Organization repo direct collaborators check'
+
+    @classmethod
+    def setUpClass(cls):
+        """Initialize the check object with configuration settings."""
+        cls.config.add_evidences(
+            [
+                ReportEvidence(
+                    'org_direct_collaborators.md',
+                    'permissions',
+                    DAY,
+                    'Organization repo direct collaborators report.'
```

Comment:
```suggestion
                    'Repository organization collaborators report.'
```

As above, since this will appear in the TOC you can see about dynamically adding the hosting service to the description.  Up to you.